src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,11 +4,16 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    public static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
         // TODO: add log message about method completeOrder was called
+        logger.info("Method completeOrder was called with userID: " + userId);
```

Comment:
The log message in the 'completeOrder' method could be more informative by using parameterized logging instead of string concatenation. This approach is more efficient and avoids unnecessary computation when the logger level is higher than INFO.